Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.6
+import PackageDescription
+
+let package = Package(
+    name: "BKMoneyKit",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "BKMoneyKit",
+            targets: ["BKMoneyKit"]),
+    ],
+    targets: [
+        .target(
+            name: "BKMoneyKit",
+            resources: [
+                .copy("BKMoneyKit.bundle/CardLogo/amex@2x.png"),
```

Comment:
I wonder if just making this `process("BKMoneyKit.bundle/CardLogo") would process all resources in that folder 🤔

Reply:
I think that would work. You can also use `.process` with individual items.

Reply:
I will just point to the `BKMoneyKit.bundle` as everything in there are just images

Reply:
Just tested an it compiles OK with just pointing to `BKMoneyKit.bundle` ✅